client/input/input.go

```diff
@@ -41,7 +41,9 @@ func GetPassword(prompt string, buf *bufio.Reader) (pass string, err error) {
 // If the input is not recognized, it returns false and a nil error.
 func GetConfirmation(prompt string, r *bufio.Reader, w io.Writer) (bool, error) {
 	if inputIsTty() {
-		fmt.Fprintf(w, "%s [y/N]: ", prompt)
+		if _, err := fmt.Fprintf(w, "%s [y/N]: ", prompt); err != nil {
```

Comment:
This could lead to undesired behaviour (e.g. `/dev/stderr` may be closed and this would cause the program to fail)